components/Typography/tests/unit/UIFont+MaterialTypographyTests.m

```diff
@@ -15,6 +15,7 @@
 #import <XCTest/XCTest.h>
 #import "MDCTypographyMockTraitEnvironment.h"
 #import "MaterialTypography.h"
+#import "MaterialApplication.h"
```

Comment:
Please add this dependency in the Cocoapods and Bazel files.

Reply:
Done. added to both.

BTW, It doesn't look like any other components do this for the unit tests in the Cocoapods.

Reply:
That's a shame. It's good to Include What You Use. ;-)